pkg/env/env.go

```diff
@@ -52,19 +52,30 @@ func NewEnv(ctx context.Context, log *logrus.Entry) (Interface, error) {
 		return nil, err
 	}
 
-	if strings.ToLower(os.Getenv("RP_MODE")) == "int" {
-		log.Warn("running in int mode")
-		return newInt(ctx, log, im, clientauthorizer.NewARM(log), clientauthorizer.NewAdmin(log))
-	}
-
 	for _, key := range []string{
 		"MDM_ACCOUNT",
 		"MDM_NAMESPACE",
+		"ADMIN_API_CA_BUNDLE_PATH",
+		"ADMIN_API_CLIENT_CERT_COMMON_NAME",
```

Comment:
Put this in in cmd/aro/rp.go wrapped by `if _, ok := _env.(env.Dev); !ok` so it doesn't trigger for aro-monitor.  We already take this approach in cmd/aro/monitor.go.  It's disgusting but we'll fix it out later.